src/js/components/DataFilters/DataFilters.js

```diff
@@ -99,6 +106,13 @@ export const DataFilters = ({ drop, children, heading, ...rest }) => {
               })}
           </Heading>
           {!controlled && clearControl}
+          {layer && (
```

Comment:
Small comment about the Heading with size="small" above (not changed in this PR but just noticing it) in relation to this being BETA and the upcoming HPE typography changes. We are going to steer away from having to apply `size` on Heading and just leverage the theme definitions. For this, we should:
- Remove size now in prep for brand updates
- Leave, but note that once the next theme version is released we should remove size="small" from the heading
- Add as theme-able thing (but this is probably a bit overkill / opening an unnecessary can of worms

Reply:
I'm assuming there are other places, such as Calendar's Heading that would be affected. So, I'm inclined to be consistent with the current master branch and update them all together. I believe this is option two in the list above.